src/condor_starter.V6.1/starter_v61_main.cpp

```diff
@@ -233,6 +233,14 @@ printClassAd( void )
 		dprintf(D_ALWAYS, "WARNING: Initializing plugins returned: %s\n", e.getFullText().c_str());
 	}
 
+
+    // Advertise the proxy/cache used by each file transfer plug-in.
+    auto proxy_by_method = ft.GetProxyByMethodMap();
+    for( auto i : proxy_by_method ) {
+        printf( "%s_proxy = \"%s\"\n", i.first.c_str(), i.second.c_str() );
+    }
```

Comment:
Now that we have c++ 17 everywhere, this could be written as

```
for (auto &[method, proxy] : ft.GetProxyByMethodMap()) {
      printf("%s_proxy = \"%s\"\n", method.c_str(), proxy.c_str());
      }
```